### ISSUE TITLE
Got exception running under Idea 9.0.4

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#238]
```
After installing the plugin through Idea plugin manager, creating a test file 
and running a test I got an exception :

"
Error during dispatching of 
java.awt.event.MouseEvent[MOUSE_RELEASED,(482,76),absolute(473,67),button=1,modi
fiers=Button1,clickCount=1] on frame0: 
com.intellij.execution.testframework.sm.runner.SMTRunnerConsoleProperties.<init>
(Lcom/intellij/execution/configurations/RuntimeConfiguration;Ljava/lang/String;L
com/intellij/execution/Executor;)V
java.lang.NoSuchMethodError: 
com.intellij.execution.testframework.sm.runner.SMTRunnerConsoleProperties.<init>
(Lcom/intellij/execution/configurations/RuntimeConfiguration;Ljava/lang/String;L
com/intellij/execution/Executor;)V
    at com.google.jstestdriver.idea.TestRunnerState.execute(TestRunnerState.java:162)
    at com.intellij.execution.impl.DefaultJavaProgramRunner.doExecute(DefaultJavaProgramRunner.java:98)
    at com.intellij.execution.runners.GenericProgramRunner$1.run(GenericProgramRunner.java:95)
    at com.intellij.execution.impl.ExecutionManagerImpl$1.run(ExecutionManagerImpl.java:129)
    at com.intellij.execution.impl.ExecutionManagerImpl.compileAndRun(ExecutionManagerImpl.java:138)
    at com.intellij.execution.runners.GenericProgramRunner.execute(GenericProgramRunner.java:119)
    at com.intellij.execution.runners.GenericProgramRunner.execute(GenericProgramRunner.java:71)
    at com.intellij.execution.ExecutionUtil.executeConfiguration(ExecutionUtil.java:77)
    at com.intellij.execution.ExecutorRegistryImpl$ExecutorAction.actionPerformed(ExecutorRegistryImpl.java:193)
    at com.intellij.openapi.actionSystem.impl.ActionButton.a(ActionButton.java:146)
    at com.intellij.openapi.actionSystem.impl.ActionButton.a(ActionButton.java:122)
    at com.intellij.openapi.actionSystem.impl.ActionButton.processMouseEvent(ActionButton.java:282)
    at java.awt.Component.processEvent(Component.java:6028)
    at java.awt.Container.processEvent(Container.java:2041)
    at java.awt.Component.dispatchEventImpl(Component.java:4630)
    at java.awt.Container.dispatchEventImpl(Container.java:2099)
    at java.awt.Component.dispatchEvent(Component.java:4460)
    at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4574)
    at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4238)
    at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4168)
    at java.awt.Container.dispatchEventImpl(Container.java:2085)
    at java.awt.Window.dispatchEventImpl(Window.java:2475)
    at java.awt.Component.dispatchEvent(Component.java:4460)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:599)
    at com.intellij.ide.IdeEventQueue.f(IdeEventQueue.java:627)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:504)
    at com.intellij.ide.IdeEventQueue.b(IdeEventQueue.java:405)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:369)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:269)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:184)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:174)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:169)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:161)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
"
```

Original issue reported on code.google.com by `d.alvare...@gtempaccount.com` on 10 May 2011 at 8:32

**Comment by GoogleCodeExporter** [comment]
```
This bug is fixed in JsTestDriver IDEA plugin v1.3 or later.
```

Original comment by `Sergey.S...@gmail.com` on 28 Jun 2011 at 2:32
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
I am using JSTestDriver 1.3 with Idea 9.0.4 and I am getting this same error.
```

Original comment by `sleepymu...@gmail.com` on 9 Sep 2011 at 10:39
